src/utils/constraints.js

```diff
@@ -11,14 +11,17 @@ export const login = {
 };
 
 export const signUp = {
```

Comment:
You should add a presence validator and even validate the content of the `gender` field.